docs/project/library-servicing.md

```diff
@@ -6,10 +6,12 @@ This document provides the steps necessary after modifying a library in a servic
 
 If a library's source project sets `<IsPackable>true</IsPackable>` a package is generated. If the library's source project doesn't set `<IsPackable>true</IsPackable>`, the library likely is part of the shared framework (to check for that, look into the `NetCoreAppLibrary.props` file). If it is, then there is nothing that needs to be done here.
 
-## Determine PackageVersion
+## Determine ServiceVersion
 
 When you make a change to a library & ship it during the servicing release, the `ServicingVersion` must be bumped. This property is found in the library's source project. It's also possible that the property is not in that file, in which case you'll need to add it to the library's source project and set it to 1. If the property is already present in your library's source project, just increment the servicing version by 1.
 
+The packages are not generated by default in servicing. You'll need to set `<GeneratePackageOnBuild>true</GeneratePackageOnBuild>` in the source project.
```

Comment:
This conflicts with the message in L7. Can you please consolidate this line with L7?

Reply:
are we gonna remove the IsPackable from the source project in servicing ?

Reply:
No. That property controls if a project is packable but not if a package should be created. For that we use `<GeneratePackageOnBuild>`. The `<IsPackable>` property shouldn't be touched at all in servicing.